docs/conf.py

```diff
@@ -40,9 +40,9 @@ def setup(app):
 autoapi_dirs = ['../robottelo']
 source_suffix = '.rst'
 master_doc = 'index'
-exclude_patterns = ['_build', 'pytest/*']
+exclude_patterns = ['_build', 'pytest/*', 'logs/']
 nitpicky = False
-autodoc_default_options = {'members': None, 'undoc-members': None}
+autodoc_default_options = {'members': None, 'undoc-members': None, 'autoapi_keep_files': True}
```

Comment:
Gonna leave this in, because it's very useful when you're trying to debug docs failures on files that get deleted when autoapi fails.